GentApp/Views/SubscriptionsPage.xaml

```diff
@@ -5,10 +5,44 @@
     xmlns:local="using:GentApp.Views"
     xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
     xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:interactivity="using:Microsoft.Xaml.Interactivity"
+	xmlns:core="using:Microsoft.Xaml.Interactions.Core"
     mc:Ignorable="d"
-    Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+    DataContext="{Binding CompaniesViewModelInstance, Source={StaticResource Locator}}">
 
-    <StackPanel>
-		<TextBlock >Your subscriptions:</TextBlock>
-    </StackPanel>
+	<interactivity:Interaction.Behaviors>
```

Comment:
kaje da nie in één command doen tegelijk?

Reply:
In feite wel, maar ik gebruik die LoadSubscriptionsCommand ook ergens anders (CompanyDetailsPage.xaml)